Client/Client.Qt/Widgets/TitleWidget.cpp

```diff
@@ -178,22 +178,32 @@ BioButton::BioButton(QWidget* parent, bool) : CaptionButton(parent)
     _popup          = std::make_unique<PopupWidget>(this);
     _settingsWidget = std::make_unique<SettingsWidget>();
 
+    ///
+
+    _dropDownMenu = std::make_unique<DropDownMenu>(this);
+
     setClickCallback([=]() {
```

Comment:
Try to avoid capturing all objects by [=], here for example you could just send __dropDownMenu_

Reply:
Already fixed